app/services/state_file/state_information_service.rb

```diff
@@ -144,11 +150,11 @@ def state_intake_class_names
         schema_file_name: "NCIndividual2024v0.7.zip",
         state_name: "North Carolina",
         submission_builder_class: SubmissionBuilder::Ty2024::States::Nc::NcReturnXml,
-        survey_link: "",
-        tax_payment_info_url: "",
-        tax_payment_url: "",
+        survey_link: "https://codeforamerica.co1.qualtrics.com/jfe/form/SV_1MM0vBfZ5N2OMLA",
+        tax_payment_info_url: "https://www.ncdor.gov/file-pay/pay-individual-income-tax",
+        tax_payment_url: "https://www.ncdor.gov/file-pay/pay-individual-income-tax",
         tax_refund_url: "https://eservices.dor.nc.gov/wheresmyrefund/SelectionServlet",
-        department_of_taxation: "",
+        department_of_taxation: "https://www.ncdor.gov/",
```

Comment:
This was a mistake in the jira ticket - the value should be the name of the Department (North Carolina Department of Revenue)